src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -500,11 +502,12 @@ def build(self, notification_uuid: str | None = None) -> SlackBlock:
         if not self.issue_details or (
             self.recipient and self.recipient.actor_type == ActorType.TEAM
         ):
-            payload_actions, action_text, color = build_actions(
-                self.group, project, text, color, self.actions, self.identity
+            payload_actions, action_text, has_action = build_actions(
```

Comment:
build_action and has_action are talking about two different things. Right?

Reply:
yep.